src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractGroupBatchAdapterRouter.kt

```diff
@@ -40,32 +42,35 @@ abstract class AbstractGroupBatchAdapterRouter<T> : MessageRouter<T> {
     }
 
     override fun subscribe(callback: MessageListener<T>, vararg attributes: String): SubscriberMonitor? {
-        return groupBatchRouter.subscribe({ consumerTag: String, message: MessageGroupBatch ->
-                callback.handle(consumerTag, buildFromGroupBatch(message))
+        return groupBatchRouter.subscribeWithManualAck({ deliveryMetadata: DeliveryMetadata, message: MessageGroupBatch, _ ->
```

Comment:
```suggestion
        return groupBatchRouter.subscribe({ deliveryMetadata: DeliveryMetadata, message: MessageGroupBatch, _ ->
```
Use method with the same name and signature because this class is adapter